samtranslator/model/stepfunctions/generators.py

```diff
@@ -129,20 +129,28 @@ def to_cloudformation(self):  # type: ignore[no-untyped-def]
             )
 
         if self.role and self.policies:
-            raise InvalidResourceException(
-                self.logical_id, "Specify either 'Role' or 'Policies' property and not both."
-            )
+            raise InvalidResourceException(self.logical_id, "Specify 'Role' or 'Policies' or neither property.")
         if self.role:
             self.state_machine.RoleArn = self.role
-        elif self.policies:
-            if not self.managed_policy_map:
+        else:
+            if self.policies and not self.managed_policy_map:
                 raise Exception("Managed policy map is empty, but should not be.")
-
+            if not self.policies:
+                self.policies = [
+                    {
+                        "Version": "2012-10-17",
+                        "Statement": [
+                            {
+                                "Action": ["Lambda:InvokeFunction"],
+                                "Effect": "Allow",
+                                "Resource": "arn:aws:lambda:us-west-2:534568764167:function:SomethingDoesNotMatter*",
+                            }
+                        ],
+                    }
+                ]
```

Comment:
Why are we adding an arbitrary policy?

Reply:
The connector doesn't work if you just leave it empty.

Reply:
🤔